code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -571,6 +573,48 @@
 	category = "Summons"
 	limit = 1
 
+//Spell loadouts datum, list of loadouts is in wizloadouts.dm
+/datum/spellbook_entry/loadout
+	name = "Standard Loadout"
+	cost = 10
+	category = "Standard"
+	refundable = FALSE
+	buy_word = "Summon"
+	var/list/items_path = list()
+	var/list/spells_path = list()
+	var/destroy_spellbook = FALSE //Destroy the spellbook when bought, for loadouts containing non-standard items/spells, otherwise wiz can refund spells
+
+/datum/spellbook_entry/loadout/GetInfo()
+	var/dat = ""
+	dat += "<b>[name]</b>"
+	if(cost > 0)
+		dat += " Cost:[cost]<br>"
+	else
+		dat += " No Cost<br>"
+	dat += "<i>[desc]</i><br>"
+	return dat
+
+/datum/spellbook_entry/loadout/Buy(mob/living/carbon/human/user, obj/item/spellbook/book)
+	if(destroy_spellbook)
+		var/response = alert(user, "The [src] loadout cannot be refunded once bought. Are you sure this is what you want?", "No refunds!", "No", "Yes")
+		if(response == "No")
+			return FALSE
+		to_chat(user, "<span class='notice'>[book] crumbles to ashes as you acquire its knowledge.</span>")
+		qdel(book)
+	else if(items_path.len)
+		var/response = alert(user, "The [src] loadout contains items that will not be refundable if bought. Are you sure this is what you want?", "No refunds!", "No", "Yes")
```

Comment:
no `The [src]`; this will come out as "The the thing".

Reply:
Loadout names are capitalised so they dont get 'the' appended by code when using [src]